website/views/manage_carousel.py

```diff
@@ -0,0 +1,120 @@
+from django.shortcuts import render, redirect
```

Comment:
I don't remember if we agreed on using PEP8 for .py files, but normally imports should be ordered alphabetically.